masschallenge/django-accelerator
# [AC-6264] Criterion Option Spec "option" field cannot be set to "blank" in admin

#### Changes introduced in [AC-6264](https://masschallenge.atlassian.net/browse/AC-6264):
- add blank=True to the the option field in the criterion option spec model

#### How to test
- checkout to this branch and run migrate
- on run accelerate
- navigate to `/admin/accelerator/criterionoptionspec/`
- You should be able to click on save without seeing field is required message

You shouldn't see this:
![screen shot 2018-12-17 at 15 24 14](https://user-images.githubusercontent.com/13675851/50087268-8d9b2c80-0210-11e9-86ac-b07deafbbe05.png)
